### PR TITLE
docs: add pull request template for project

### PR DESCRIPTION
**Pull request by allencloud**
Signed-off-by: Allen Sun <allensun.shl@alibaba-inc.com>

docs: add pull request template for project

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+https://github.com/openkruise/kruise/blob/master/CONTRIBUTING.md -->
+
+### Ⅰ. Describe what this PR did
+
+
+### Ⅱ. Does this pull request fix one issue?
+<!--If that, add "fixes #xxxx" below in the next line, for example, fixes #15. Otherwise, add "NONE" -->
+
+
+### Ⅲ. Why don't you add test cases (unit test/integration test)? (你真的觉得不需要加测试吗？)
```
Comment (xiang90) [inline .github/PULL_REQUEST_TEMPLATE.md]: can you remove the Chinese explanation? thanks.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<!-- 
+Please make sure you have read and understood the contributing guidelines;
+https://github.com/openkruise/kruise/blob/master/CONTRIBUTING.md -->
+
+### Ⅰ. Describe what this PR did
```
Comment (Fei-Guo) [inline .github/PULL_REQUEST_TEMPLATE.md]: Did -> does

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (Fei-Guo) [reply .github/PULL_REQUEST_TEMPLATE.md]: How about this?

List the added test cases (unit test/integration test) if any, please explain if no tests are needed.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+
+### Ⅱ. Does this pull request fix one issue?
+<!--If that, add "fixes #xxxx" below in the next line, for example, fixes #15. Otherwise, add "NONE" -->
```
Comment (Fei-Guo) [inline .github/PULL_REQUEST_TEMPLATE.md]: If that -> If so,

**Comment by allencloud** [comment]
@xiang90 @Fei-Guo Thanks for your feedback. PTAL

**Comment by xiang90** [comment]
lgtm

**Pull request merged**